packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -4019,6 +4019,46 @@ Expected request body:
 { "requiredProperty": "foo", "nullableProperty": null }
 ```
 
+### Type_Property_Optional_BooleanLiteral_getAll
+
+- Endpoint: `get /type/property/optional/boolean/literal/all`
+
+Expected response body:
+
+```json
+{"property": doc}
+```
+
+### Type_Property_Optional_BooleanLiteral_getDefault
+
+- Endpoint: `get /type/property/optional/boolean/literal/default`
+
+Expected response body:
+
+```json
+{}
+```
+
+### Type_Property_Optional_BooleanLiteral_putAll
+
+- Endpoint: `put /type/property/optional/boolean/literal/all`
+
+Expected request body:
+
+```json
+{ "property": true }
+```
+
+### Type_Property_Optional_BooleanLiteral_putDefault
+
+- Endpoint: `put /type/property/optional/boolean/literal/default`
+
+Expected request body:
+
+```json
+{}
+```
```

Comment:
why do we expect empty body in Type_Property_Optional_BooleanLiteral_putDefault?

Reply:
ok, got it, it's just `?`